packages/commons/src/auth/authData.ts

```diff
@@ -3,11 +3,22 @@ import { z } from "zod";
 
 export const AuthJWTToken = z.object({
   organizationId: z.string().uuid(),
-  "user-roles": z.string(),
-  role: z.string(),
-  sub: z.string().uuid(),
+  "user-roles": z
```

Comment:
just a consideration: some fields are optional because we handle different types of tokens.
It might be worth to define one TS type for each use case to remove ambiguities (and at the same time clean up the model and logics for our tokens).

Point for a future PR btw.

cc @beetlecrunch